src/tests/shipments_api.test.ts

```diff
@@ -70,6 +76,36 @@ describe('Shipments API', () => {
         },
       })
 
+      expect(res.errors).not.toBeUndefined()
+      expect(res.errors).not.toBeEmpty()
+
+      if (res.errors == null || res.errors.length === 0) {
+        return
+      }
```

Comment:
This is annoying, I wish the assertions on lines 79 and 80 meant this wasn't necessary but TypeScript didn't agree.